src/EA.Weee.Web/Areas/Producer/Views/Shared/_CheckAnswersContentPartial.cshtml

```diff
@@ -157,6 +160,26 @@
                 </td>
             </tr>
         }
+        <tr class="govuk-table__row">
+            <th scope="col" class="govuk-table__header govuk-!-text-align-left" style="width: 28%;">Selling technique(s) that you use</th>
+            <td class="govuk-table__cell" colspan="2">
+                @switch(Model.EeeData.SellingTechnique.ToSellingTechniqueType())
+                {
+                    case SellingTechniqueType.DirectSellingToEndUser:
+                        sellingTechniqueType = Html.DisplayNameFor(m => m.EeeData.SellingTechnique.IsDirectSelling).ToString();
+                        break;
+                    case SellingTechniqueType.IndirectSellingToEndUser:
```

Comment:
@stevelee-civica This should be done in code. There is EnumHelper that I think should do what we need, does that not work? The model method can be unit tested, this cant...

Reply:
updated and added unit tests